CHANGELOG.md

```diff
@@ -373,57 +373,350 @@ All notable changes to this project will be documented in this file. See [standa
 
 * **plugin-conversation:** Send kms on join ([#2568](https://github.com/webex/webex-js-sdk/issues/2568)) ([0960a66](https://github.com/webex/webex-js-sdk/commit/0960a66830559d01656b04cfc0642e7ff820e041))
 
-### [2.37.1](https://github.com/webex/webex-js-sdk/compare/v2.37.0...v2.37.1) (2023-01-17)
+### [2.37.1](https://github.com/webex/webex-js-sdk/compare/v3.0.0-beta.14...v2.37.1) (2023-01-17)
```

Comment:
What do we want to do about the CHANGELOG? Which changes do we want to keep?

Reply:
I think we need not worry about the change log as we have the change logs for the respective releases in their respective branches. I remember that's what Marcin mentioned in the space as well. As long as we keep beta and master branches, changelog shouldn't matter